.github/workflows/macos.yml

```diff
@@ -49,7 +49,7 @@ jobs:
 
           # Compile bootloader
           cd bootloader
-          python waf distclean all
+          python waf --no-universal2 distclean all
```

Comment:
I guess we will wait for `macos-11.0` public availability before being able to enable unversal builds.
As stated in the [doc](https://github.com/actions/virtual-environments#available-environments):

> The macOS 11.0 virtual environment is currently provided as a private preview only.
> The "macos-latest" YAML workflow label still uses the macOS 10.15 virtual environment.